src/hb-ot-hmtx-table.hh

```diff
@@ -330,6 +358,22 @@ struct hmtxvmtx
     hb_blob_ptr_t<V> var_table;
   };
 
+  unsigned get_advance (const hb_subset_plan_t *plan,
```

Comment:
I don't like this subset-specific `get_advance` here. If anything, name it properly. You notice I named all `get_advance` functions like `with_variations_unscaled` etc.

Reply:
I changed it to `get_new_gid_advance_unscaled`. It gets the advance from mtx_map in subset_plan when there're variations, otherwise it calls get_advance_without_var_unscaled